lib/features.js

```diff
@@ -88,6 +88,14 @@ const features = {
                  'capabilities or host volume mounts.',
     defaults: false,
     module: require('./features/dind')
+  },
+
+  websocketServer: {
+    title: 'Docker Exec Websocket Server',
+    description: 'Runs another process alongside the original executed process' + 
+                 'and pipes the result through websockets.',
```

Comment:
These docs should probably focus on how you can use this to fake "SSH" into the container...